src/utils/safe-message-guards.ts

```diff
@@ -1,12 +1,10 @@
-import { SafeMessageListItemType } from '@/store/safeMessagesSlice'
-import type { SafeMessage, SafeMessageListPage, SafeMessageDateLabel } from '@/store/safeMessagesSlice'
+import { SafeMessageListItemType } from '@gnosis.pm/safe-react-gateway-sdk'
+import type { SafeMessageListItem, SafeMessage, SafeMessageDateLabel } from '@gnosis.pm/safe-react-gateway-sdk'
 
-export const isSafeMessageListDateLabel = (
-  item: SafeMessageListPage['results'][number],
-): item is SafeMessageDateLabel => {
+export const isSafeMessageListDateLabel = (item: SafeMessageListItem): item is SafeMessageDateLabel => {
   return item.type === SafeMessageListItemType.DATE_LABEL
 }
 
-export const isSafeMessageListItem = (item: SafeMessageListPage['results'][number]): item is SafeMessage => {
+export const isSafeMessageListItem = (item: SafeMessageListItem): item is SafeMessage => {
```

Comment:
`isSafeMessage`?

Reply:
I don't want to change these as they mirror that of the transaction list.